backend/model/mixins/reindex_top_containers.rb

```diff
@@ -1,6 +1,9 @@
 module ReindexTopContainers
 
   def reindex_top_containers
+    # get out if we aren't part of a resource tree
+    # TODO: look at this - we probably need to touch this record's top containers at least
+    return unless self.root_record_id
```

Comment:
@marktriggs this caused problems when running tests with lonely archival_objects.